src/authentification/views.py

```diff
@@ -26,7 +26,7 @@ def post(self, request, *args, **kwargs):
         # here we make sure that a post request won't trigger a subscription in case allow_signup is False
         if not bool(settings.ALLOW_SIGNUP):
             return redirect('signup')
-        if not hasattr(settings, "MAILJET_API_KEY") or not settings.MAILJET_API_KEY:
+        if not settings.DEBUG and not (getattr(settings, "MAILJET_API_KEY") and settings.MAILJET_API_KEY):
```

Comment:
But here we have a case not tackled, don't we ?
let's say we are on our machine, with DEBUG = True and we want to see the email in the logs. How do we do that ?

Reply:
I have to check that. Sounds weird to me since the fucntion _EmailMessage_ is called afterwards. Therefore, if DEBUG=True, _EmailMessage_ is never reached. And the logs cannot display anything then

Reply:
ça m'a fait découvrir ce bug: #14